lib/charms/layer/jenkins/plugins.py

```diff
@@ -39,16 +39,18 @@ def install(self, plugins):
         plugins = plugins or ""
         plugins = plugins.split()
         plugins = self._get_plugins_to_install(plugins)
-
+        configured_plugins = hookenv.config()["plugins"].split()
         host.mkdir(
             paths.PLUGINS, owner="jenkins", group="jenkins", perms=0o0755)
-        existing_plugins = set(glob.glob("%s/*.jpi" % paths.PLUGINS))
+        existing_plugins = set(glob.glob("%s/*.?pi" % paths.PLUGINS))
```

Comment:
I think it'd be better to only allow jpi and hpi rather than any character followed by pi.

Reply:
Addressed in 68328ba547e2e7d8ea49436ae70470b050ad8867.